app/editor/src/features/admin/reports/ReportList.tsx

```diff
@@ -45,7 +45,11 @@ export const ReportList: React.FC = () => {
                 reports.filter(
                   (i) =>
                     i.name.toLocaleLowerCase().includes(value) ||
-                    i.description.toLocaleLowerCase().includes(value),
+                    i.description.toLocaleLowerCase().includes(value) ||
+                    i.owner?.username.toLocaleLowerCase().includes(value) ||
```

Comment:
We can now search for the report by the owner. 
Sorting however doesn't work correctly.  I need to enhance the FlexboxTable component.